File: src/tool/spring-boot-generator/output/test/test/src/main/java/com/test/pojo/entity/StationLine.java
package com.test.pojo.entity;

import com.test.pojo.entity.StationLine;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import javax.validation.constraints.NotNull;



@ApiModel("车站线路")
@Getter
@Setter
public class StationLine @extends {

    @ApiModelProperty("中心点距离")
    private Float centDistance;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("行别")
    private Integer direction;

    @ApiModelProperty("终端距离")
    private Float endDistance;

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("删除标记")
    @NotNull
    private Integer isDelete;

    @ApiModelProperty("线路编号")
    @NotNull
@NotNull
    private Integer lineId;

    @ApiModelProperty("操作人")
    @NotNull
@NotNull
    private Integer operator;

    @ApiModelProperty("排序编号")
    @NotNull
@NotNull
    private String sortNum;

    @ApiModelProperty("开始距离")
    @NotNull
@NotNull
    private Float startDistance;

    @ApiModelProperty("车站编号")
    @NotNull
@NotNull
@NotNull
    private Integer stationId;

    @ApiModelProperty("状态")
    @NotNull
@NotNull
@NotNull
    private Integer status;

    @ApiModelProperty("更新时间")
    @NotNull
@NotNull
@NotNull
    private Date updateTime;

}

File: src/tool/spring-boot-generator/output/railway/db/stationLine.js
let stationLine = {
	name: "stationLine",
	description: "车站线路",
	columns: {
		centDistance: {
			name: "centDistance",
			description: "中心点距离",
			nullable: true,
			type: {
				name: "float"
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		direction: {
			name: "direction",
			description: "行别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		endDistance: {
			name: "endDistance",
			description: "终端距离",
			nullable: true,
			type: {
				name: "float"
			}
		},
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		lineId: {
			name: "lineId",
			description: "线路编号",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		operator: {
			name: "operator",
			description: "操作人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		sortNum: {
			name: "sortNum",
			description: "排序编号",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		startDistance: {
			name: "startDistance",
			description: "开始距离",
			nullable: true,
			type: {
				name: "float"
			}
		},
		stationId: {
			name: "stationId",
			description: "车站编号",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		status: {
			name: "status",
			description: "状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		}
	}
};

let columnsArray = [
	"centDistance",
	"createTime",
	"direction",
	"endDistance",
	"id",
	"isDelete",
	"lineId",
	"operator",
	"sortNum",
	"startDistance",
	"stationId",
	"status",
	"updateTime"
];

function create() {
  return {};
}

stationLine.columnsArray=columnsArray;
stationLine.create=create;
module.exports=stationLine;